lib/ansible/modules/net_tools/nmcli.py

```diff
@@ -1051,7 +1051,11 @@ def create_connection(self):
             cmd = self.create_connection_bridge()
         elif self.type == 'vlan':
             cmd = self.create_connection_vlan()
-        return self.execute_command(cmd)
+        if cmd:
+            return self.execute_command(cmd)
```

Comment:
Seems inconsistent, there are both `cmd = ...` and `return self.execute_command(cmd)` above, maybe we can unify that?

Reply:
I am not sure, I understand your comment correctly. 
I am checking cmd if self.type does not matched to any of given parameters. same in modify_connections.

Reply:
Sorry,  I meant we could get rid of all `return self.execute_command(cmd)` in the elif's block above and use just the return here on this line instead? If you look at all the elif's, some of them return directly and some of them just assign `cmd`. So I thought we could unify that. Not something you caused of course.